docker-for-windows/edge-release-notes.md

```diff
@@ -11,6 +11,17 @@ This page contains information about Docker Desktop Edge releases. Edge releases
 For information about Stable releases, see the [Stable release notes](release-notes.md). For Docker Desktop system requirements, see
 [What to know before you install](install.md#what-to-know-before-you-install).
 
+## Docker Desktop Community 2.5.2
+2020-11-24
+
+> [Download](https://desktop.docker.com/win/edge/50063/Docker%20Desktop%20Installer.exe)
+
+### New
+- Use of three digit version number.
```

Comment:
If we're advertising delta updates in Mac we should put them here too.

Reply:
Done!